app/templates/rating/index.html

```diff
@@ -0,0 +1,71 @@
+{% extends 'layouts/base.html' %}
+
+{% block javascript %}
+    {% assets 'resources_js' %}<script type="text/javascript" src="{{ ASSET_URL }}"></script>{% endassets %}
+{% endblock %}
+
+{% block content %}
+    <div class="ui stackable grid container">
+        <div class="sixteen wide tablet twelve wide computer centered column">
+            <h2 class="centered ui header">
+                Ratings and Reviews
+                <div class="sub header">
+                    View current ratings and reviews
+                </div>
+            </h2>
+
+
+        <!--<div class="ui menu">
```

Comment:
What's the purpose of this commented out part? I think this is particular to the 'All Resources' view and not for ratings/reviews